zoho.go

```diff
@@ -50,14 +53,30 @@ func (s *ZohoService) AddRow(tableUri string, columnValues map[string]string) (*
 
 	var addedRows ZohoAddRowResponse
 
-	err := s.sendAPIRequest(columnValues, tableUri, addRow, &addedRows)
+	err := s.sendAPIRequest(columnValues, tableUri, addRowAction, &addedRows)
 	if err != nil {
 		return nil, errors.Wrap(err, errMessage)
 	}
 
 	return &addedRows, nil
 }
 
+// ImportCSV import a bulk of rows as CSV
+func (s *ZohoService) ImportCSV(tableUri, csvData string, config map[string]string) (*ZohoAddRowResponse, error) {
```

Comment:
Should we not potentially have a check if the CSV that is being passed in here as a string is actually a valid csv file?

Reply:
wouldn't it also be better to provide it as a byte array?